ios-base/Networking/TargetType+Base.swift

```diff
@@ -49,4 +49,18 @@ extension TargetType {
   public func requestParameters(parameters: [String: Any]) -> Task {
     return .requestParameters(parameters: parameters, encoding: JSONEncoding.default)
   }
+
+  public func multipartData(from parameters: [String: Any]) -> [MultipartFormData] {
+    return parameters.map { (key: String, value: Any) -> MultipartFormData? in
+      let value = value
+      let key = "user[\(key)]"
+      if let value = value as? Data {
+        return MultipartFormData(provider: .data(value), name: key)
+      } else if let data = "\(value)".data(using: String.Encoding.utf8, allowLossyConversion: false) {
+        return MultipartFormData(provider: .data(data), name: key)
```

Comment:
Just a question: Does `MultipartFormData` takes care of building arrays or dictionaries the right way?
For arrays it should be the `key[]='value 1' key[]='value 2'` and so on.

Reply:
That's a really good question, not sure though, I need to test it somehow

Reply:
Cool, please also test the dictionary case. Let me know.